keep/api/models/db/tenant.py

```diff
@@ -36,4 +36,4 @@ class TenantInstallation(SQLModel, table=True):
     tenant_id: str = Field(foreign_key="tenant.id")
     bot_id: str
     installed: bool = False
-    tenant: Optional[Tenant] = Relationship(back_populates="installations")
+    tenant: Optional[Tenant] = Relationship(back_populates="installations")
```

Comment:
?

Reply:
@shahargl Please clarify what is the question here

Reply:
why this line changed?

Reply:
Idk, it seems to be the same. I wasn't changing anything in this section.